data/sql_updates/reports_pac_party.sql

```diff
@@ -1,8 +1,8 @@
-drop materialized view if exists ofec_reports_pacs_parties_mv_tmp;
+drop materialized view if exists ofec_reports_pacs_parties_mv_tmp cascade;
```

Comment:
Just double checking: this `cascade` won't have any unintended side effects, will it?